.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+By filing an Issue, you are expected to comply with the elementary code of conduct: https://elementary.io/code-of-conduct
+
+Please fill out this template with all the information you have. We can't do much without a detailed description of what you've encountered. Please do your best!
+
+Please note that this tracker is only for bugs and feature requests. Please try these locations if you have a question or comment:
+
+  https://elementaryos.stackexchange.com/
+  https://www.reddit.com/r/elementaryos/
+
+Please read and follow these tips:
+https://elementary.io/docs/code/reference#be-prepared-to-provide-more-information
+
+Lastly, be sure to preview your issue before saving. Thanks!
+-->
+
+## Prerequisites
+- [ ] I have searched open and closed issues for duplicates.
```

Comment:
Do we need this anymore since GitHub includes search when creating a title now?

Reply:
I don't think we need it. I know some third-party clients might not search for issues ahead of time (i.e. an Android app or whatever) but that doesn't really seem like what we'd be catering to with issue templates anyway.

Reply:
I would mainly include it to make people think twice. It kind of reiterate that people should really do this. 

Also these relates issues list only looks at the title and as we all know there are usually many ways to word things. When searching for existing issues I usually try 2 or 3 terms. 

In some cases it's useful to search through all the issues of the elementary org. For example when people open an issue in the nightlight indicator repo which is a duplicate of something in the nightlight settings repo. But I'm not sure how to explain that and if that applies often enough.